ansible/playbooks/k8s-all.yml

```diff
@@ -1,3 +1,7 @@
 - import_playbook: common.yml
+- import_playbook: cri.yml
+- import_playbook: iptables.yml
```

Comment:
Instead of adding these here they can now be added to the `common.yml` file, use the `master.yml` as a reference example.